content/components/hidden/react/draft.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Hidden
+description: Use Hidden to responsively hide or show content in narrow, regular and wide viewports.
+reactId: hidden
+reactStatus: draft
+---
+
+import ReactComponentPage from '~/src/components/react-component-page'
+import { graphql } from "gatsby"
+
+export const query = graphql`
+  query HiddenDraftQuery($componentId: String! = "hidden", $parentPath: String! = "/components/hidden", $status: String! = "draft") {
+    ...ReactComponentInfo
+  }
+`
+
+export default function Layout({data, children}) {
+  return <ReactComponentPage data={data}>{children}</ReactComponentPage>
+}
+
+The `Hidden` component is a utility component that helps hide or show content/components in different viewports.
+
+<Note>Use Hidden only to control behaviour in a responsive manner. It does not take any `sx` props.</Note>
+
+## Examples
```

Comment:
I think we just want to render a "default" example and rely on Storybook for everything else. For example: https://primer.style/components/action-list/react/beta